style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a:hover {
+  color: #172b4d;
+}
+
+.html,
+body {
+  font-family: Poppins, sans-serif;
+}
+
+.blue {
+  color: #6070ff;
+}
+
+.header_nav {
+  display: flex;
+  min-height: 10vh;
+  padding: 10px 20px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header_logo {
+  font-size: 22px;
+  font-weight: 700;
+  line-height: 20px;
+}
+
+.header_logo a {
+  text-decoration: none;
+}
+
+.header_menu {
+  padding: 3px 5px;
+}
+
+.menu_icon {
+  width: 24px;
+  height: auto;
+}
+
+.headline {
+  padding: 25px;
+  display: flex;
+  min-height: 50vh;
+  background-size: contain;
+  flex-direction: column;
+  background-repeat: space;
```

Comment:
- [ OPTIONAL] it would be better if you use `background-repeat: no-repeat; and background-size: cover`. this way your back will be a single image and cover all your section background.

Reply:
@sajid-munawar Thanks a lot I really appreciate your optional requirement. Can I Edit them before merging my pull request although the project was approved?